package.json

```diff
@@ -26,14 +26,19 @@
   },
   "main": "index.js",
   "devDependencies": {
-    "chai": "^3.2.0",
-    "mocha": "^5.2.0",
-    "request": "^2.88.0",
-    "standard": "^12.0.1"
+    "ed25519-supercop": "^1.2.0",
+    "events.once": "^2.0.2",
+    "get-port": "git+https://github.com/davidmarkclements/get-port.git",
```

Comment:
can you point to a SHA or registry release?

Reply:
I'm working with sindresorhus here: https://github.com/sindresorhus/get-port/pull/36 so hopefully we can update this to an actual npm release. However failing that, I'll update to the SHA

Reply:
ok, now reply yet, please set to a SHA so we can proceed

Reply:
this was merged last night, https://github.com/sindresorhus/get-port/pull/36 - updating to the 5.1.0 release